packaging/hipify-clang.txt

```diff
@@ -7,25 +7,53 @@ install(DIRECTORY @HIPIFY_INSTALL_PATH@/include DESTINATION bin)
 #############################
 # Packaging steps
 #############################
-set(CPACK_SET_DESTDIR TRUE)
-set(CPACK_INSTALL_PREFIX @CPACK_PACKAGING_INSTALL_PREFIX@)
-set(CPACK_PACKAGE_NAME "hipify-clang")
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "hipify-clang: a clang-based tool to translate CUDA source code into portable HIP C++ automatically")
+set(CPACK_PACKAGE_NAME @PROJECT_NAME@)
 set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc.")
-set(CPACK_PACKAGE_CONTACT "Mankov Evgeny <Evgeniy.Mankov@amd.com>")
-
-set(CPACK_PACKAGE_VERSION @LLVM_VERSION_MAJOR@.@LLVM_VERSION_MINOR@.@LLVM_VERSION_PATCH@)
 set(CPACK_PACKAGE_VERSION_MAJOR @LLVM_VERSION_MAJOR@)
 set(CPACK_PACKAGE_VERSION_MINOR @LLVM_VERSION_MINOR@)
 set(CPACK_PACKAGE_VERSION_PATCH @LLVM_VERSION_PATCH@)
+#TODO : Change mail id to standard <package.suppport@amd.com>
+set(CPACK_PACKAGE_CONTACT "Mankov Evgeny <Evgeniy.Mankov@amd.com>")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "hipify-clang: a clang-based tool to translate CUDA source code into portable HIP C++ automatically")
+set(CPACK_SET_DESTDIR TRUE)
+set(CPACK_INSTALL_PREFIX @CPACK_PACKAGING_INSTALL_PREFIX@)
 set(CPACK_GENERATOR "TGZ;DEB;RPM")
-set(CPACK_BINARY_DEB "ON")
-set(CPACK_BINARY_RPM "ON")
-set(CPACK_RPM_PACKAGE_ARCHITECTURE "${CMAKE_SYSTEM_PROCESSOR}")
+
+#Make proper version for appending
+#Default Value is 99999, setting it first
+set(ROCM_VERSION_FOR_PACKAGE "99999")
+if(DEFINED ENV{ROCM_LIBPATCH_VERSION})
```

Comment:
Which component provides this env. variable?

Reply:
This is defined by the CI builds env. When User builds locally these are undefined & hence high number for version is replaced